pmem: build-images: too many requests: You have reached your pull rate limit

```
9:28:32 Sending build context to Docker daemon  62.01MB
19:28:33 Step 1/48 : ARG LINUX_BASE=debian:buster-slim
19:28:33 Step 2/48 : FROM ${LINUX_BASE} AS build
19:28:33 toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
19:28:35 Makefile:98: recipe for target 'build-image' failed
19:28:35 make[1]: *** [build-image] Error 1
19:28:35 make[1]: Leaving directory '/home/jenkins/workspace/go/src/github.com/kata-containers/tests/pmem-csi'
19:28:35 [pmem_test.sh:77] ERROR: make build-images
```

--- a/integration/pmem/pmem_test.sh
+++ b/integration/pmem/pmem_test.sh
@@ -74,7 +74,7 @@ setup_pmem_csi() {
 	git clone https://github.com/intel/pmem-csi/
 	pushd pmem-csi
 
-	make build-images
+	make BUILD_ARGS="--build-arg LINUX_BASE=public.ecr.aws/debian/debian:buster-slim" build-images
 	make push-images
 
 	kubectl label node $(hostname) storage=pmem

